README.md

```diff
@@ -46,7 +46,7 @@ docker run -p 3000:3000 -v $PWD:/usr/src/app korerorero-mouth-shapes:latest
 
 Use the docker instructions above. It's a whole thing to set up this environment on your local machine.
 
-Visit <http://localhost:3000/> and you should see the Rhubarb version installed.
+Once you have the docker image running, visit <http://localhost:3000/> and you should see the Rhubarb version installed.
 
```

Comment:
I've added some words to make it clear that although the URL includes localhost, that the server needs to be running on a docker instance. cc @mermop